src/dosbox.cpp

```diff
@@ -652,7 +652,7 @@ void DOSBOX_Init()
 	secprop->AddInitFunction(&PAGING_Init);
```

Comment:
Typo in the commit message (first line, "inidicate")

Reply:
Thanks, fixed.